docs/application/dotnet/guides/internationalization/localization.md

```diff
@@ -0,0 +1,252 @@
+
+1. Add the `RESX` resource files that will be used to store all the text used in the your application.
+![local_resx](media/local_resx.png)
+![local_appresources_resx1](media/local_appresources_resx1.png)
+
+2. Change the string visibility from internal to public. Select your `resx` file and click **Properties**. In the Configuration Properties, change Custom Tool to PublicResXFileCodeGenerator as shown in the following image:
+![local_appresources_resx2](media/local_appresources_resx2.png)
+![local_appresources_resx3](media/local_appresources_resx3.png)
+
+3. Add language-specific resource files, which must follow a specific naming convention and use the same filename as the base resources file.
+![local_appresources_resx4](media/local_appresources_resx4.png)
+
+- Use name of texts with String type in the `RESX` files in your user interface code.
+
+**C# file**
+```cs
+var speedLabel = new Label ();
+var maximumLabel = new Label ();
+
+speedLabel.Text = AppResources.Speed;
+maximumLabel.Text = AppResources.Maximum;
+```
+
+- Use name of texts with String type in the `RESX` files in the your `xaml` code.
+
+**XAML file**
+
+```xml
+<Application
+    x:Class="Speedmeter.App"
+    xmlns="http://xamarin.com/schemas/2014/forms"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:local="clr-namespace:Speedmeter"
+    xmlns:resx="clr-namespace:Speedmeter.Resx;"
+    xmlns:w="clr-namespace:Tizen.Wearable.CircularUI.Forms;assembly=Tizen.Wearable.CircularUI.Forms">
+    <Application.MainPage>
+        <w:CirclePage>
+            <CirclePage.Content>
+                <AbsoluteLayout>
+                    <Label
+                        x:Name="SpeedLabel"
+                        AbsoluteLayout.LayoutBounds="0.5, 0.3"
+                        AbsoluteLayout.LayoutFlags="PositionProportional"
+                        Style="{StaticResource LabelStyle-Base}"
+                        Text="{x:Static resx:AppResources.Speed}" />
+                    <Label
+                        x:Name="AverageLabel"
+                        AbsoluteLayout.LayoutBounds="0.2, 0.7"
+                        AbsoluteLayout.LayoutFlags="PositionProportional"
+                        Style="{StaticResource LabelStyle-Base}"
+                        Text="{x:Static resx:AppResources.Average}" />
+```
+
+## Display the correct Language
+To display the correct language, you must implement additional code in your project to determine which language the user has selected.
+Please refer to following Xamarin.Forms article.
+
+For more information, see [Display the correct Language](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/localization/text?tabs=vswin#displaying-the-correct-language)
+
+1. Define an interface to use your `DependencyService`:
+
+```cs
```

Comment:
Global comment : To make a code block as a sub item of a list, 3 spaces are needed before each line of code block like below.

```
1. Define an interface

   ```cs
   public interface ILocalize

```

Reply:
done